documentation/how-to/tracking.en.md

```diff
@@ -5,89 +5,91 @@ tx_slug: documentation_how-to_tracking
+If did not explicitly terminate your tracking session (for example, you closed the app), QField allows you to continue where you left off.
+A prompt will appear asking whether to **resume the last session** or **start a new one**.
+
+- **Resuming** a session for a line or polygon layer will continue adding vertices to the feature from the previous session.
+This allows you to simply continue across app restarts.
+
+- **Starting a new session** will discard the incomplete feature from the previous session and begin a new one.
 
 ## Tracking underway
 
-Once a tracking session has been setup, a badge will appear in the side dashboard
-legend next to the layer(s) against which tracking is being recorded.
+Once a tracking session has been set up, a badge will appear in the side dashboard legend next to the layer(s) against which tracking is being recorded.
 
 !![](../assets/images/tracking-badge.png)
 
 The features created by the tracking session are saved on every vertex recorded.
-A rubberband line overlayed onto the map is attached to each tracking session,
-allowing you to glance at what has been recorded.
+A rubberband line overlayed onto the map is attached to each tracking session, allowing you to glance at what has been recorded.
 
 If the layer linked to a tracking session supports the M dimension, QField will store
 the time passed since the first vertex recorded in each vertex's M value.
 
-!!! note
+!!! tip
     You can simultanously record multiple trackings sessions across several layers.
 
-!!! note
-    While the tracking session is active the layer is still editable, but the tracked feature can't be edited or deleted until the tracking session is stopped over the legend again.
-
 ## Terminate tracking
 
-To terminate tracking, simply open the side dashboard. From there, long-press
-on a legend layer linked to an active tracking session and select the
-*Stop tracking* button.
+1. Open the side dashboard and press long on the tracking layers layer
+2. Select the *Stop tracking* button.
 
 !![](../assets/images/tracking-stop.png)
 
-## Predefined Project Tracking Sessions
+## Automatic Tracking Sessions
 
-This functionality enables one or more position tracking sessions to automatically start upon project load. Users will be presented with a feature form popup as the tracking session begins to fill in attributes. If the vector layer used to track has the "Hide Form on Add Feature" mode selected, the feature form will be skipped.
+This functionality enables one or more position tracking sessions to automatically start upon project load.
+The feature form of the layer will immediately open asking you for the attributes.
+If the tracked vector layer has the "Hide Form on Add Feature" mode selected, the feature form will be skipped.
 
 !![Hide Form on Added Feature](../assets/images/hide-form-on-add-feature.png)
 
-### Configure a Project Tracking Session
+### Configuration of an automatic Tracking Session
```

Comment:
See above about project tracking session being IMHO a better name. Also, title case is not respected.